Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3
+
+RUN pip3 install operator-courier
```

Comment:
Could operator-courier be installed from the local dir `pip3 install .`  It allows to build containers directly from git source automatically.

Reply:
done